core/quantity/src/Parser.ts

```diff
@@ -333,22 +333,33 @@ export class Parser {
           }
         } else {
           // not processing a number
-          if (isOperator(charCode)) {
+          const isCharOperator = isOperator(charCode);
+          const isSpacer = charCode === format.spacerOrDefault.charCodeAt(0) && charCode !== QuantityConstants.CHAR_SPACE;
+
+          if (isSpacer && i > 0 && i < str.length - 1) {
+            const prevCharCode = str.charCodeAt(i - 1);
+            const nextCharCode = str.charCodeAt(i + 1);
+            if (isCharOperator && nextCharCode !== QuantityConstants.CHAR_SPACE && prevCharCode !== QuantityConstants.CHAR_SPACE) {
+              // ignore spacer if it's not at the start or end, not whitespace, and is not in between whitespace
+              continue;
+            }
+          }
+
+          if(wipToken.length === 0 && charCode === QuantityConstants.CHAR_SPACE){
```

Comment:
```suggestion
          if (wipToken.length === 0 && charCode === QuantityConstants.CHAR_SPACE) {
```

Reply:
I just noticed that this file goes back and forth between whether or not it has space around the parentheses for `if` statements. I _believe_ that spaces are supposed to be there for all if/for/while, etc. statements, but I'm not 100% sure of that.

Reply:
We should be automating enforcement of any style rules that we care about (I advocate using [prettier](https://prettier.io/). Nitpicking over them on PRs is a waste of valuable developer time.